src/main/java/com/fasterxml/jackson/databind/util/EnumResolver.java

```diff
@@ -126,7 +126,27 @@ public static EnumResolver constructUsingToString(DeserializationConfig config,
                 config.isEnabled(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS));
     }
 
-    /**
+
```

Comment:
Need to add Javadoc that includes `@since 2.15` to indicate when method was added.